tripy/tests/integration/test_iota.py

```diff
@@ -54,11 +54,11 @@ def _compute_ref_iota(self, dtype, shape, dim):
             ((2, 3, 4), 2),
         ],
     )
-    def test_iota(self, dtype, shape, dim):
+    def test_iota(self, dtype, shape, dim, compile_fixture):
         if dim:
-            output = tp.iota(shape, dim, dtype[1])
+            output = compile_fixture(tp.iota, shape, dim, dtype[1])
         else:
-            output = tp.iota(shape, dtype=dtype[1])
+            output = compile_fixture(tp.iota, shape, dtype=dtype[1])
```

Comment:
This is weird. `dim=None` should work, so we only need one branch here:
```py
output = compile_fixture(tp.iota, shape, dim, dtype[1])
```

Reply:
@parthchadha my mistake, I was thinking of other ops. `dim=None` doesn't make sense for iota since it can only operate along one dimension. Maybe we should just update the test to use `dim=0` instead of `None`?